manifests/common/plumgrid.pp

```diff
@@ -0,0 +1,16 @@
+# Common class used by Plumgrid Neutron Plugin.
+
+class openstack::common::plumgrid {
+  # forward all ipv4 traffic
+  # this is required for the vms to pass through the gateways public interface
+  sysctl::value { 'net.ipv4.ip_forward': value => '1' }
+
+  # ifc_ctl_pp needs to be invoked by root as part of the vif.py when a VM is powered on
+  file { '/etc/sudoers.d/ifc_ctl_sudoers':
+    ensure  => file,
+    owner   => root,
+    group   => root,
+    mode    => '0440',
+    content => 'nova ALL=(root) NOPASSWD: /opt/pg/bin/ifc_ctl_pp *\n',
+  }
```

Comment:
ifc_ctl and ifc_ctl_pp are command line tools that provide the equivalent to 'brctl addif' for iovisor. It is used from nova's vif.py [https://github.com/openstack/nova/blob/master/nova/virt/libvirt/vif.py] in the PLUMgrid specific section.